packages/autorest.typescript/test/rlcIntegration/generated/bodyStringRest/package.json

```
@@ -42,7 +42,7 @@
     "@azure/identity": "^4.0.1",
     "@azure-tools/test-recorder": "^3.0.0",
     "autorest": "latest",
-    "c8": "^8.0.0",
+    "c8": "^9.0.0",
```

```suggestion
    "c8": "^9.1.0",
```
I thought it's 9.1.0 that we are trying to upgrade to ?

---

You are right. It should be 9.1.0